sample-input/sph-factors/assembly/compute-mgxs.py

```diff
@@ -0,0 +1,58 @@
+import copy
+import numpy as np
+import openmc.mgxs
+from infermc.materialize import differentiate_mgxs_lib
```

Comment:
I don't have any strong preference for keeping/removing the assembly SPH problem, so I'll just defer to you on whether you want it to be kept it in; if it is kept in, perhaps there could be a try/catch around the import of InferMC and an error message printed out informing the user of the need for InferMC to run the input file?